src/views/router_views/accounts_submitted/accounts_submitted.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block main_content %}
+    {{ govukPanel({
+        titleText: "Accounts submitted",
+        html: "Your reference number<br><strong>" + transactionId + "</strong>"
+    }) }}
+
+    {{ govukTable({
+        firstCellIsHeader: false,
+        rows: [
+                [
+                    {
+                        text: "Company number" 
+                    },
+                    {
+                        text: companyProfile.companyNumber
+                    } 
+                ],
+                [
+                    {
+                        text: "Company name" 
+                    },
+                    {
+                        text: companyProfile.companyName
+                    } 
+                ],
+                [
+                    {
+                        text: "Payment received" 
+                    },
+                    {
+                        text: "£" + payment
+                    } 
+                ]
+            ]   
+    })
+
+    }}
+
+    <p class="govuk-body">We have sent a confirmation email to <span class="govuk-!-font-weight-bold">{{ userEmail }}</span></p>
+
+    <h2 class="govuk-heading-m">What happens next</h2>
+
+    <p class="govuk-body">We will create your presenter accounts file. We'll email you to confirm if it's accepted.</p>
+    <p class="govuk-body">If it is rejected, we'll email to tell you what to do next. If you paid to file the accounts, we will refund the fee.</p>
+
+    <h2 class="govuk-heading-m">File more package accounts</h2>
+    <a href="https://upload-accounts-prototype-28bdf03329ea.herokuapp.com/v1/06-choose-your-package-accounts">File more package accounts for {{companyProfile.companyName}} ({{companyProfile.companyNumber}})</a>
```

Comment:
These links should be the real ones, not prototype. This should link to the upload screen.